icepyx/core/granules.py

```diff
@@ -473,7 +462,7 @@ def place_order(
 
         return self.orderIDs
 
-    def download(self, verbose, path, session=None, restart=False):
+    def download(self, verbose, path, restart=False):
```

Comment:
It looks like this is the change that conflicts with `icepyx/core/query.py:1113` (see my most recent comment for details)

Reply:
Nice catch! I'll approve and merge #521 - thanks for finding the issue and submitting a fix (the failing CI tests are behind authentication, so they fail because forks cannot access the secrets). I will send you an invite to the icepyx dev team here so you can edit the PR directly, too.